### ISSUE TITLE
Pendiente generación de PR para corrección

### ISSUE THREAD

**Issue by ivanfranco502** [yomelabancoyque/Proyecto1#1]
@yomelabancoyque  Queda pendiente la generación del PR para su corrección. Cualquier consulta sobre cómo realizarlo avisame y lo vemos por acá.

**Comment by yomelabancoyque** [comment]
hola ivan, quería consultarte..tengo hasta el 18 para entregarlo?
saludos

**Comment by ivanfranco502** [comment]
@yomelabancoyque exacto, la idea es que en la medida que vayas teniéndolo listo lo presentes. Sobre todo para evitar que se nos junten todas las correcciones para los mismos días.

**Comment by ivanfranco502** [comment]
@yomelabancoyque no pudiste revisarlo?
